### ISSUE TITLE
[New first run onboarding] [Accessibility] - Loading animation not announced with voice over

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#29783]
## Steps to reproduce
1. Turn on voice over from iOS settings.
1. Install FF.
1. Tap on “Continue” button.
1. Listen.

#### **Actual Result:**
Loading animation is announced with voice over.

#### **Expected Result:**
Loading animation is not announced with voice over.

Device & build information

* Devices: iPhone 16+(26)
* Build version: v9000 (62338)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-13728)

**Comment by data-sync-user** [comment]
➤ Razvan Litianu commented:

Winnie Teichmann  , Jeffrey Guerrero , should we announce the loading Firefox logo? If so, what should it say?

**Comment by data-sync-user** [comment]
➤ Winnie Teichmann commented:

Razvan Litianu I don’t think we need to announce the logo but we should announce that we are loading so that the user knows what is going on.

**Comment by data-sync-user** [comment]
➤ Razvan Litianu commented:

Thanks for the response, Winnie Teichmann . I think it makes sense. The only thing, we’re not technically loading anything, so my dilemma now is what should it say? Will “Loading” be ok?

**Comment by data-sync-user** [comment]
➤ Winnie Teichmann commented:

Razvan Litianu yes, “Loading” sounds good to me.

**Comment by data-sync-user** [comment]
➤ Razvan Litianu commented:

Winnie Teichmann, after our offline conversation where we discussed that we’re not technically loading anything, is it okay to not announce it for now? Or is it ok to proceed with “Loading”?

**Comment by data-sync-user** [comment]
➤ Razvan Litianu commented:

Diana Andreea Barladeanu I’m moving this to QA for visibility. But after some discussions with Winnie, the loader is really short and it’s not worth it to be announced.

**Comment by data-sync-user** [comment]
➤ Miruna Curtean commented:

Validated on v9000(62151), with iPhone 14 Pro Max(26). After the “Continue” button is announced, the animation is skipped and the “Skip” button is announced next. 

!RecordIt-A1EB8E7E-5BD1-4297-B228-2FEF20D2D7FD.mp4|width=396,alt="RecordIt-A1EB8E7E-5BD1-4297-B228-2FEF20D2D7FD.mp4"!
